src/components/TableList/TableList.test.js

```diff
@@ -31,9 +35,11 @@ describe("TableList", () => {
   it("displays all data from redux store", () => {
     const store = mockStore(dataDump);
     const wrapper = mount(
-      <Provider store={store}>
-        <TableList />
-      </Provider>
+      <Router history={newHistory}>
```

Comment:
What's going on with this change? Does this have to be here for a valid test? If so then it'll need to go on every test, meaning we will need to create a 'const wrapper = generateComponent(options)` method that can be shared between tests.